app/Http/Controllers/TestController.php

```diff
@@ -5,12 +5,14 @@
 use App\Http\Requests;
 use App\Http\Controllers\Controller;
 use App\User;
+use Auth;
 
 class TestController extends Controller
 {
 	public function test()
 	{
-		return response()->json(['dd' => 'name']);
+		$usr =  Auth::guard('api')->user();
+		return $usr;
```

Comment:
What should I understand from the word `$usr` ? Please give meaningful name eg `$user`